src/Mod/Path/PathScripts/PathOpGui.py

```diff
@@ -1375,7 +1375,7 @@ def Create(res):
     that is created in each operations Gui implementation."""
     FreeCAD.ActiveDocument.openTransaction("Create %s" % res.name)
     try:
-        obj = res.objFactory(res.name, obj=None, parentJob=res.job)
+        obj = res.objFactory(res.name, obj=None)#, parentJob=res.job)
```

Comment:
Sorry for being late to the party. This is not the best way of avoiding the issue and this change breaks creation of ops when there are multiple jobs in a single file.

The much better, and correct way of dealing with the issue is add the `parentJob` named parameter to all the `Create(..)` functions of the Ops in _FreeCAD_Turning_Addon_ .

FYI: the additional parameters for `Create(..)` came in ~ a year ago with commit #4797

Reply:
@WarrenKCash ping